<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxtMainWindow</class>
 <widget class="QDialog" name="TxtMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>190</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Txt Final</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-image: url(:/images/pdfback.jpg);
    background-repeat: no-repeat;
}

QWidget{
font: 11pt &quot;MS Shell Dlg 2&quot;;
}

QRadioButton{
	font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>1. Please select a TXT file</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="rightMargin">
         <number>30</number>
        </property>
        <item>
         <widget class="QLineEdit" name="editTxtFile">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectTxt">
          <property name="text">
           <string>TXT</string>
          </property>
          <property name="icon">
           <iconset resource="bf.qrc">
            <normaloff>:/images/txt.png</normaloff>:/images/txt.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnConvert">
       <property name="text">
        <string>Convert to SNB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>Read SNB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpload">
       <property name="text">
        <string>Upload to Bambook</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 85, 0);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Txt Final has been integreated to Bambook Factory</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="bf.qrc"/>
 </resources>
 <connections/>
</ui>
